docs/2.10.0/docs/canton/usermanual/console.rst

```diff
@@ -85,7 +85,13 @@ TLS and Authorization
 ^^^^^^^^^^^^^^^^^^^^^
 
 For production use cases, in particular if the Admin API is not just bound to localhost, we recommend to enable
-:ref:`TLS <tls-configuration>` with mutual authentication.
+:ref:`TLS <tls-configuration>` with mutual authentication. If an API is secured using certificates that are
+singed by a system-wide trusted CA and TLS configuration is otherwise empty, TLS can be enabled
```

Comment:
```suggestion
:ref:`TLS <tls-configuration>` with mutual authentication. If an API is secured using certificates that are
signed by a system-wide trusted CA and the TLS configuration is otherwise empty, TLS can be enabled
```